sky/engine/platform/BUILD.gn

```diff
@@ -97,7 +97,6 @@ source_set("platform") {
     "fonts/harfbuzz/HarfBuzzFaceSkia.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.h",
-    "fonts/linux/FontPlatformDataLinux.cpp",
```

Comment:
Ah yeah, that's probably it. //buildroot/build/BUILDCONFIG.gn uses set_sources_assignment_filter.